stage1/init/common/pod.go

```diff
@@ -604,7 +604,7 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		return errwrap.Wrap(errors.New("failed to link service want"), err)
 	}
 
-	if err = writeAppReaper(p, appName.String()); err != nil {
+	if err = writeAppReaper(p, appName.String(), common.RelAppRootfsPath(appName), execStartString); err != nil {
```

Comment:
Instead of "execStartString", you can give "binPath" here: it is the binary (absolute path) without parameters or quotes.